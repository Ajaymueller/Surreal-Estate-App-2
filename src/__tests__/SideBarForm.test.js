import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import SideBarForm from '../Components/SideBarForm';
import { MemoryRouter } from 'react-router-dom';
import "@testing-library/jest-dom/extend-expect";

const props = {
    handleChange: jest.fn(), 
    handleSearch: jest.fn(), 
    query: "random", 
};

describe("SideBarForm", () => {
    it("component renders with correct props", () => {
        const { asFragment } = render (
        <MemoryRouter><SideBarForm {...props}/></MemoryRouter> )
        expect(asFragment).toMatchSnapshot();
    });
    xit("renders a form", () => {
        const { getByTestId } = render (
        <MemoryRouter><SideBarForm {...props}/></MemoryRouter> )
        const form = getByTestId("sideBarForm");
        expect(form).toBeInTheDocument();
    });
});