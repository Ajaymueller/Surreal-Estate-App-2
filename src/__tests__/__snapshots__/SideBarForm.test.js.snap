exports[`SideBarForm component renders with correct props 1`] = `[Function]`;
